jsx/InfoPanel.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Display a message in an information panel.
+ *
+ * @param {object} props - React props
+ *
+ * @return {JSX}
+ */
+function InfoPanel(props) {
+    return (
+        <div className="alert alert-info"
+           style={{
+             backgroundColor: '#BEDFFF',
```

Comment:
Hi @driusan just something to keep in mind that inline styling has worse performance than using className. See https://reactjs.org/docs/faq-styling.html#are-inline-styles-bad

Reply:
That's interesting because it's the exact opposite of the advice that react gave when it was first released and they said to use inline styling so that components are self-contained.. but we're not hitting any performance bottlenecks yet.